### PR TITLE
3d toaster animation

### PR DESCRIPTION
**Pull request by apu52**
I have added a new project on 3D toaster animation & have added a gif file with it.
I have made it using HTML & CSS
Please take a look at my PR & please merge it.

### CODE REVIEW

**Comment by apu52** [comment]
@l-white Please take a review of my PR & merge it.

**Comment by LaurelineP** [comment]
Gif has a fail to start audio that could be avoided :)

**Comment by apu52** [comment]
@l-white Please take a review of my PR & merge it. I have added the correct GIF file & have updated the include.js file

**Comment by apu52** [comment]
@l-white Please take a review of my PR & merge it. I have added the correct GIF file & have updated the include.js file

**Comment by apu52** [comment]
@l-white Please take a review of my PR & merge it. I have added the correct GIF file & have updated the include.js file

**Comment by apu52** [comment]
@l-white Please take a review of my PR & merge it. I have added the correct GIF file & have updated the include.js file

File: include.js [hunk 1]
```diff
@@ -407,6 +407,13 @@ let cards = [
     author: 'Sneha Das',
     githubLink: 'https://github.com/iamsnehadas'
   },
+  {
+    artName: '3D toaster Animation', 
+    pageLink: './Art/https://github.com/apu52/index.html', 
+    imageLink: './Art/https://github.com/apu52/3D Toaster.gif', 
+    author: 'Arpan Chowdhury', 
+    githubLink: 'https://github.com/https://github.com/apu52'
```
Comment (LaurelineP) [inline include.js:415]: # Feedbacks on include js content
- [ ] invalid githubLink url > `https://github.com/https://github.com/apu52`
- [ ] invalid imageLink > this does not convey with a folder name you added a link which in incorrect
- [ ] invalid pageLink > this does not convey with a folder name you added a link which in incorrect
Please check the example on the readme.md file and correctly modify your PR 
--- 
also note that your file must not have any space character in their names
--> this should includes modifications based on your folder structure and file renaming

File: Art/apu52/3D Toaster.gif
Comment (LaurelineP) [inline Art/apu52/3D Toaster.gif:1]: # Feedback - gif file name
Please rename the file so there are no space - use letters, digits, hyphens or underscore instead

**Review by LaurelineP — CHANGES_REQUESTED** [review]
Both your html and css file seems to override a prior contributions
-  Please let us know what you want to do 
- Please adjust your files and folder
For any guidance please refer to the README.md file

**Pull request closed without merging**